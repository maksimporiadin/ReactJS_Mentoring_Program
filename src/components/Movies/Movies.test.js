import 'jsdom-global/register';
import React from 'react';
import { Movies } from './index';
import { shallow } from 'enzyme';

describe('Movies', () => {
    let wrapper;
    const defaultProps = {
        movies: []
    };

    beforeEach(()=>{
        jest.resetAllMocks();
    });

    it('is renders without movies', () => {
        wrapper = shallow(<Movies {...defaultProps} />)
        expect(wrapper).toMatchSnapshot();
    });

    it('is renders without movies', () => {
        const props = {
            movies: [{id: 1, movie: {}}]
        };
        wrapper = shallow(<Movies {...props} />)
        expect(wrapper).toMatchSnapshot();
    });
});