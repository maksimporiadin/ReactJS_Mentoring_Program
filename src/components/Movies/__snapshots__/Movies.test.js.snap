// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies is renders without movies 1`] = `
<div
  className="movies-container"
>
  <NoFilmsFound />
</div>
`;

exports[`Movies is renders without movies 2`] = `
<div
  className="movies-container"
>
  <Movie
    key="0"
    movie={
      Object {
        "id": 1,
        "movie": Object {},
      }
    }
  />
</div>
`;
